packages/tools/testing-tools/src/flow/saga/mock-command-implementation.ts

```diff
@@ -0,0 +1,21 @@
+import { MockedCommandImplementation, SagaContext } from '../../types'
+import { makeAssertions, SagaAssertionsNode } from './make-assertions'
+
+export const mockCommandImplementation = (
+  context: SagaContext,
+  aggregateName: string,
+  type: string,
+  implementation: MockedCommandImplementation
+): SagaAssertionsNode => {
+  const { environment } = context
+
+  if (environment.isExecuted()) {
+    throw Error(
+      `Command implementation cannot be mocked if the test was executed.`
```

Comment:
```suggestion
      `Command implementation cannot be mocked if the test is finished.`
```
?
/cc @EugeniyBurmistrov

Reply:
The original phrase and the edit mean different things. The edit is indeed correct if the message's indended meaning was "is already finished" and not "was invoked and possibly still runs".